config/e2e.yml

```diff
@@ -0,0 +1,28 @@
+boolTestKey: true
+numberTestKey: 1
+stringTestKey: 'string'
+
+rollbar:
+  accessToken: 'ROLLBAR_ACCESS_TOKEN'
+  env: 'ROLLBAR_ENV'
+
+jwt:
+  token: 'JWT_TOKEN'
+
+communication:
+  services: ['twilio', 'sendgrid']
+
+twilio:
+  verify:
+    accountSid: 'AC.TWILIO_ACCOUNT_SID'
+    authToken: 'TWILIO_AUTH_TOKEN'
+  messaging:
+    messagingServiceSid: 'TWILIO_MESSAGING_SERVICE_SID'
+
+sendgridApiKey: 'SG.SENDGRID_API_KEY'
+
+mongoDb:
+  uri: 'mongodb://localhost:27017/boilerplate-mern-e2e'
+
+server:
+  port: '8080'
```

Comment:
As mentioned above, this can be removed as this will get picked up by default (config/default.yml) and this has no effect here.